ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/DownloadFile.kt

```diff
@@ -137,7 +135,8 @@ class DownloadFile(
         Util.delete(partialFile)
         Util.delete(completeFile)
         Util.delete(saveFile)
-        mediaStoreService.deleteFromMediaStore(this)
+
+        Util.scanMedia(context, saveFile)
```

Comment:
This might look funny, but is ok, because the Uri of the deleted file is still stored in the File Class